src/contact-information/model/model.cto

```diff
@@ -0,0 +1,19 @@
+namespace org.accordproject.contactinformation
+
+import org.accordproject.cicero.contract.* from https://models.accordproject.org/cicero/contract.cto
+import org.accordproject.cicero.runtime.* from https://models.accordproject.org/cicero/runtime.cto
+import org.accordproject.timezone.TimeZone from https://deploy-preview-118--accordproject-models.netlify.app/timezone.cto
```

Comment:
Those should point to the public models.

Reply:
Yes, I will update once the models PR is merged.